feat: 드롭다운 연결 / 후기-메시지모달 연결

JiminN2 opened the pull request:
## 🔎 작업 내용
- 기능에서 어떤 부분이
- 구현되었는지 설명해주세요
- 내역필터 드롭다운 연결했고 드롭버튼 UI 수정했습니다. 시안대로 하려 했는데 똑같이 못하겠습니다ㅜ 다른 의견 있으면 알려주세요 
- 취소 모달 api 연결 했습니다
- 후기모달에서 확인 메시지 모달 연결 했습니다

## 📜 간단한 코드 설명 및 사용설명서


## 📷 결과물 (image , gif ..)
![image](https://github.com/user-attachments/assets/144031cc-9dec-43a1-99ac-4d40da7e71a0)
![image](https://github.com/user-attachments/assets/a4aa14a1-5bed-4f21-90ab-93bc696ab912)
![image](https://github.com/user-attachments/assets/e478d8a7-d8dd-4bb7-bfe9-ddda245b52ee)




### 👀 공유포인트 및 이슈
-빌드에러 곧 수정하겠습니다

JiminN2 committed:
feat: 예약취소 api 연결

JiminN2 committed:
feat: dropdown ui 1차 수정

JiminN2 committed:
Merge branch 'develop' into KAN-96--reviewdrop

JiminN2 committed:
Merge branch 'develop' into KAN-96--reviewdrop

JiminN2 committed:
feat: 확인메세지모달 연결

JiminN2 committed:
chore: 주석삭제

jinyunkim commented:
![image](https://github.com/user-attachments/assets/f457b1ab-49dc-4986-92fe-f4e0ac7fc4e9)
이 부분 제가 리팩토링 할게여!
오아 감사합니다..

jinyunkim commented:
빌딩 에러 난거 체크해주세요!

jinyunkim commented on app/(features)/(user)/myreservations/_components/Modal/ReservationInfo.tsx:
```diff
@@ -28,12 +28,21 @@ const ReservationInfo = ({ reservation }: ReservationInfoProps) => {
-        {activity && <Image src={activity.bannerImageUrl} priority layout="fill" objectFit="cover" alt="배너 이미지" />}
+      <div className="overflow relative h-[90px] w-[83px] max-w-full sm:h-[100px] sm:w-[100px] md:h-[126px] md:w-[126px]">
+        {activity && (
+          <Image
+            src={activity.bannerImageUrl}
+            priority
+            layout="fill"
+            objectFit="cover"
+            alt="배너 이미지"
+            className="rounded-3xl"
+          />
```
이 부분 개발자모드에서 warning 안 뜨나요??

JiminN2 replied:
앗 waring 떴던 것 같아요 수정해보겠습니다

JiminN2 committed:
import type { Metadata } from "next";
import { Inter } from "next/font/google";
import localFont from "next/font/local";
import { QueryProvider } from "./QueryProvider";
import "./styles/globals.css";

const pretendardStd = localFont({
  src: "./assets/fonts/PretendardStdVariable.woff2",
  display: "swap",
  variable: "--font-pretendard-std",
});

const inter = Inter({ subsets: ["latin"], display: "swap", variable: "--font-inter" });

export const metadata: Metadata = {
  title: "GlobalNomad",
  description:
    "GlobalNomad - 여행을 계획하고 다양한 활동을 쉽게 탐색하세요. Plan your trips and explore various activities with ease.",
  icons: {
    icon: "/favicon.ico",
  },
};

export default function RootLayout({
  children,
}: Readonly<{
  children: React.ReactNode;
}>) {
  return (
    <html lang="ko">
      <head />
      <body className={`${pretendardStd.variable} ${inter.variable}`}>
        <QueryProvider>{children}</QueryProvider>
      </body>
    </html>
  );
}
Merge branch 'feature/MyReservation' of https://github.com/GlobalNomads/TripNomadHub into feature/MyReservation

JiminN2 committed:
feat: status type 수정

JiminN2 committed:
fix: status 타입 수정완료

JiminN2 committed:
fix: eslint airbnb 설치

JiminN2 committed:
chore: uninstall airbnb

JiminN2 committed:
fix: eslint airbnb 제거

JiminN2 committed:
chore: airbnb hooks 다시 추가

JiminN2 committed:
fix: 주석처리

jinyunkim commented:
오.. 필터 예뻐졌네요!

JiminN2 committed:
fix: patch 타입 수정

JiminN2 committed:
fix

JiminN2 committed:
fix: modal header mobile 되돌리기

JiminN2 committed:
feat: 확인버튼 onClose 추가

JiminN2 committed:
fix: Image Waring해결

Ik5606 reviewed (approved):
수정확인했습니다! 고생하셨어요

jinyunkim reviewed (approved):
수고하셨어요!!
